src/filing/actions/receiveEdit.js

```diff
@@ -1,14 +1,66 @@
+import React from 'react'
```

Comment:
I think from a certain version of react and onwards this is no longer needed; I'm guessing our version still pre-dates that, but just want to call it out here for now.

Reply:
Tried removing it but got an error.  :+1: